src/lib/docs/AccountKit/Account.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Account
+description: An Account instance is a wrapper for Antelope blockchain accounts. It facilitates interaction with accounts by offering methods for account data retrieval and action generation.
+category: AccountKit
+published: true
+order: 2
+---
+
+# Account
+
+Each `Account` class instance represents a specific blockchain account. It offers several helper methods to facilitate interaction with that account.
+
+## Creation
+
+In most cases, the [AccountKit Factory](/docs/account-kit/account-kit-factory) will be used to generate `Account` instances. However, they can also be created manually if the relevant data is provided. Here is a basic example of how to do so:
+
+```ts
+import { Account } from "@wharfkit/contract"
+import { APIClient } from "@wharfkit/antelope"
+
+import abi from "./eosio-abi.json" // ABI for the eosio.token contract
+
+const contractArgs = {
+    abi,
+    account: "eosio.token",
+    client: new APIClient("https://jungle4.greymass.com"),
+}
+
+const contract = new Account(contractArgs)
```

Comment:
This code example needs reworked. `ContractArgs` and a bunch of other contract-related imports and variable names need to be cleaned up here.

And does the token contract need to be passed here?

Reply:
It doesn't need to be, but I'll add an example showing how you would pass it 👍